### ISSUE TITLE
Is there support for Razer Kraken?

### ISSUE THREAD

**Issue by walterMenlon** [1kc/razer-macos#683]
a while ago, i bought a Razer Kraken set off of amazon. (no idea of what the exact model is, but it's the normal one, no kitty ears. it's also a headphone jack and not USB.) 

I use a 2015 Macbook Air (Monterey) and this seems to be the only thing that works with the computer i have, yet the program doesnt seem to recognize my headphones...

If anyone has a fix for this issue, please comment!!

(keep in mind, i have no idea what any technical or programming stuff means, so if you're talking about technical stuff, please dumb it down by a lot)

**Comment by walterMenlon** [comment]
Update: i have found that they are Razer Kraken 7.1

**Comment by dhobi** [comment]
Razer macOS only supports listing and controlling devices connected through USB, sorry. 😞
